include/glow/Importer/ProtobufLoader.h

```diff
@@ -103,15 +103,36 @@ class ProtobufLoader {
   /// \returns the tensor that was registered under the name \p name.
   Tensor *getTensorByName(const std::string &name);
 
+  /// Create a new node \p name initialized with \p tensor.
+  /// The created node is also registered under \p name.
+  /// \returns the newly created node.
+  /// \pre !hasNodeByName(name)
+  Node *createAndRememberNode(
+      const std::string &name, Tensor &tensor,
+      VisibilityKind visibilityKind = VisibilityKind::Private,
+      Variable::TrainKind trainKind = Variable::TrainKind::Broadcast);
+
+  /// \returns the node that was registered with the name \p name or nullptr
+  /// if no node has been registered with this name.
+  Node *getNodeByNameOrNull(const std::string &name) const;
+
 public:
   /// \returns the node that was registered with the name \p name.
-  Node *getNodeByName(const std::string &name);
+  /// \pre hasNodeByName(name)
+  Node *getNodeByName(const std::string &name) const;
 
   /// \returns the node that was registered with the name \p name or create a
   /// new Variable node for a tensor with this name.
   Node *getOrCreateNodeByName(const std::string &name);
 
-  /// \returns True if the node that's registered using \p name exists.
+  /// Create a new node \p name initialized with \p tensor.
+  /// \returns the newly created node.
+  Node *
+  createNode(const std::string &name, Tensor &tensor,
```

Comment:
I don't like the name getNode because it is not very descriptive. The code that uses getNode is not very readable. I understand that getVariable may not be exact if we add the Load node. Can you think of another name?

Reply:
This actually follows the nomenclature used previously (getOrCreateNodeByName and co.)
I can change all the names to put variable instead, but the diff is going to be bigger.
In other words, what I am asking is should we do the renaming as a follow-up commit or do it here?

Note: I'll go with createVariable and if/when we go with the load node we could go with createVariableAndLoad (maybe?)